src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPicker.cs

```diff
@@ -0,0 +1,1375 @@
+			// since otherwise we'll get into an infinite loop of updating.
+			if (m_updatingControls)
+			{
+				return;
+			}
+
+			// We'll respond to the text change if the user has entered a valid value.
+			// Otherwise, we'll do nothing except mark the text box's contents as invalid.
+			var value = ColorConversion.TryParseInt(m_valueTextBox.Text);
+			if (!value.HasValue ||
+				value.Value < (ulong)this.MinValue ||
+				value.Value > (ulong)this.MaxValue)
+			{
+				m_isFocusedTextBoxValid = false;
+			}
+			else
+			{
+				m_isFocusedTextBoxValid = true;
+				UpdateColor(GetHsvColorFromTextBoxes(), ColorUpdateReason.HsvTextBoxChanged);
+			}
+		}
+
+		private void OnAlphaTextChanging(TextBox sender, TextBoxTextChangingEventArgs args)
+		{
+			// If we're in the process of updating controls in response to a color change,
+			// then we don't want to do anything in response to a control being updated,
+			// since otherwise we'll get into an infinite loop of updating.
+			if (m_updatingControls)
+			{
+				return;
+			}
+
+			if (m_alphaTextBox is TextBox alphaTextBox)
+			{
+				// If the user hasn't entered a %, we'll do that for them, keeping the cursor
+				// where it was before.
+				int cursorPosition = alphaTextBox.SelectionStart + alphaTextBox.SelectionLength;
+
+				char s = alphaTextBox.Text[alphaTextBox.Text.Length - 1];
+
+				if (s != '%')
+				{
+					alphaTextBox.Text = alphaTextBox.Text + "%";
+					alphaTextBox.SelectionStart = cursorPosition;
+				}
+
+				// We'll respond to the text change if the user has entered a valid value.
+				// Otherwise, we'll do nothing except mark the text box's contents as invalid.
+				string alphaString = alphaTextBox.Text.Substring(0, alphaTextBox.Text.Length - 1);
+				var alphaValue = ColorConversion.TryParseInt(alphaString);
+				if (!alphaValue.HasValue || alphaValue.Value < 0 || alphaValue.Value > 100)
+				{
+					m_isFocusedTextBoxValid = false;
+				}
+				else
+				{
+					m_isFocusedTextBoxValid = true;
+					UpdateColor(alphaValue.Value / 100.0, ColorUpdateReason.AlphaTextBoxChanged);
+				}
+			}
+		}
+
+		private void OnHexTextChanging(TextBox sender, TextBoxTextChangingEventArgs args)
+		{
+			// If we're in the process of updating controls in response to a color change,
+			// then we don't want to do anything in response to a control being updated,
+			// since otherwise we'll get into an infinite loop of updating.
+			if (m_updatingControls)
+			{
+				return;
+			}
+
+			var hexTextBox = m_hexTextBox;
+
+			// If the user hasn't entered a #, we'll do that for them, keeping the cursor
+			// where it was before.
+			if (hexTextBox.Text[0] != '#')
+			{
+				hexTextBox.Text = "#" + hexTextBox.Text;
+				hexTextBox.SelectionStart = hexTextBox.Text.Length;
+			}
+
+			// Uno Doc: This section assigning rgbValue/alphaValue was re-written in C# to avoid strange usage of functions in C++
+			//const bool isAlphaEnabled = IsAlphaEnabled();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6121259)